.github/workflows/main.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
     branches: [main]
   pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
```

Comment:
Wouldn't this make it run the entire lint, test, build workflow as well for every non-code PR change?